src/en/help-azure.md

```diff
@@ -26,7 +26,111 @@ juju update-clouds
 
 ## Credentials
 
-In order to access Azure, you will need to add some credentials for Juju to use.
+Using Juju's interactive authentication, importing Azure credentials into Juju
+is now a simple process. The only information you'll need is your Azure subscription
```

Comment:
Is the "now" relevant to people who haven't used the provider before? That seems more like a comment for release notes.